app/src/androidTest/java/com/conupods/OutdoorMaps/navigateActivitiesInstrumentedTest.java

```diff
@@ -37,7 +37,7 @@
 @RunWith(AndroidJUnit4.class)
 public class navigateActivitiesInstrumentedTest {
     private static final String TAG = "TEST_NAVIGATION";
-    private static final String[] SGW_CODES = {"Z", "MM", "Q", "LC", "GNL", "K", "EN", "MK", "CI", "V", "FG", "ER", "MI", "LD", "R", "GA", "PR", "MB", "X", "P", "GS", "TU", "S", "ET", "EV", "GM", "B", "VA", "TD", "RR", "T", "MU", "ES", "MT", "FA", "H", "D", "LB", "CL", "MN", "M", "FB"};
+    private static final String[] SGW_CODES = {"Z", "MM", "Q", "LC", "GNL", "K", "EN", "MK", "CI", "V", "FG", "ER", "MI", "LD", "R", "GA", "PR", "X", "P", "GS", "TU", "S", "ET", "GM", "B", "TD", "RR", "T", "MU", "ES", "MT", "FA", "H", "D", "LB", "CL", "MN", "M", "FB"};
```

Comment:
I may have missed this conversation why are some rooms removed. Also JMSB should be that instead of MS too maintain consistency.

Reply:
they were removed for failing tests that failed unnecessarily. Markers would be actually in place (took screenshots) and be clickable (took screenshots of that too) and yet it the tests failed on those 3 markers. Spoke with Sandra and removed them afterwards.